### PR TITLE
Feat#1284✨: Added Loadmore button

### LINKED ISSUE #1284: Add "Load more" button to created bounties in the people page
### Context
Add "Load more" button to created bounties in the people page and restrict visibility to 20 bounties. 

### Design
Match it with the design in the assigned bounties section

### Acceptance Criteria
- [ ] I've tested on Chrome
- [ ] I've submitted a screenshot or recording in my pr
- [ ] I've created mock data for the api to load more bounties
- [ ] I've created a component test that tests the "load more" button is displayed when we scroll down. Here is an [example unit test](https://github.com/stakwork/sphinx-tribes/blob/9310f49b3b17a51992dada932f4298eb9eba15ff/frontend/app/src/people/widgetViews/__tests__/AboutView.spec.tsx)

### PR DESCRIPTION
**Pull request by VividhPandey003**
## Describe your changes
Added loadmore button to created bounties 

https://github.com/stakwork/sphinx-tribes/assets/91251535/5b8394c9-4086-4b1c-8e8e-142a06a12d0a

## Issue ticket number and link

Closes #1284 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by elraphty** [comment]
@VividhPandey003 `src/pages/people/tabs/CreatedBountyLoadMore.spec.tsx` test is failing

**Comment by VividhPandey003** [comment]
> @VividhPandey003 `src/pages/people/tabs/CreatedBountyLoadMore.spec.tsx` test is failing

yeah, working on it

File: frontend/app/src/pages/people/tabs/Wanted.tsx [hunk 1]
```diff
@@ -106,7 +114,7 @@ export const Wanted = observer(() => {
       >
         {canEdit && <PostBounty widget="wanted" />}
       </div>
-      {main.createdBounties.map((w: any, i: any) => {
+      {main.createdBounties.slice(0, visibleItems).map((w: any, i: any) => {
```
Comment (elraphty) [inline frontend/app/src/pages/people/tabs/Wanted.tsx:117]: @VividhPandey003 The pagination call should be sent to the backend with `page & limit` queries, we have moved all the paginations to the backend

Comment (VividhPandey003) [reply frontend/app/src/pages/people/tabs/Wanted.tsx:117]: So basically just passing page and query parameter to getCreatedBounties fn would do it.
And so I need to remove the slice logic as well right

Comment (VividhPandey003) [reply frontend/app/src/pages/people/tabs/Wanted.tsx:117]: Okay. Thanks

**Comment by ecurrencyhodler** [comment]
New api endpoint for our backend is: https://api.people-test.sphinx.chat/

**Pull request closed without merging**